config/initializers/date_time_formats.rb

```diff
@@ -7,3 +7,4 @@
 
 Time::DATE_FORMATS[:default] = "%d %b %Y %k:%M"
 Time::DATE_FORMATS[:date_only] = "%d %b %Y"
+Time::DATE_FORMATS[:csv] = "%Y-%m-%d"
```

Comment:
nit: perhaps we should name the format after this very idiomatic format iso8601, but agains i have not rela problem with csv expect that it is indicating what format we expect in a csv rather than the format itself. not sure what is better

Reply:
have left it as :csv for now as I found it clearer?